lib/common/options.c

```diff
@@ -585,6 +590,41 @@ pcmk__print_option_metadata(const char *name, const char *desc_short,
                     _(option_list[lpc].description_short),
                 (option_list[lpc].values? _("  Allowed values: ") : ""),
                 (option_list[lpc].values? _(option_list[lpc].values) : ""));
+
+                option_list[lpc].description_short,
+                option_list[lpc].description_long?
+                    option_list[lpc].description_long :
+                    option_list[lpc].description_short,
+                (option_list[lpc].values? "  Allowed values: " : ""),
+                (option_list[lpc].values? option_list[lpc].values : ""));
```

Comment:
To reconcile this conflict, delete the first set of lines starting with description_short, and make your changes in the second set